doc/man/8/rados.rst

```diff
@@ -104,7 +110,7 @@ Pool specific commands
   List the watchers of object name.
 
 :command:`ls` *outfile*
-  List objects in given pool and write to outfile.
+  List objects in the given pool and write to outfile. Instead of --pool if --pgid will be specified, ls will only list the objects in the given PG.
```

Comment:
could you quote "--pool", "--pgid" and "ls" with backquotes?

Reply:
added.